src/controllers/expenses.js

```diff
@@ -0,0 +1,77 @@
+'use strict';
+
+const expensesService = require('../services/expenses');
+
+const getAll = async(req, res) => {
+  const expenses = await expensesService.getAll();
+
+  res.send(expenses);
+};
+
+const getOne = async(req, res) => {
+  const { expenseId } = req.params;
+
+  const foundExpense = await expensesService.getById(expenseId);
+
+  if (!foundExpense) {
+    res.sendStatus(404);
+
+    return;
+  };
+
+  res.send(foundExpense);
+};
+
+const create = async(req, res) => {
+  const newExpense = req.body;
+  const { title, userId } = newExpense;
+
+  if (!title || !userId) {
+    res.sendStatus(400);
+
+    return;
+  };
+
+  await expensesService.addOne(newExpense);
+
+  res.statusCode = 201;
+  res.send(newExpense);
+};
+
+const remove = (req, res) => {
+  const { expenseId } = req.params;
+
+  expensesService.deleteOne(expenseId);
```

Comment:
Add checking if expense is exist and only then delete it. You can call your function `expensesService.getById` to check it. And if there is no such expense, send status code 404